fhirAssembly/src/main/resources/assembly/com.vantiq.fhir.fhirConnection/sources/com/vantiq/fhir/fhirServer.json

```diff
@@ -14,7 +14,7 @@
     "requestDefaults" : {
       "contentType" : "application/fhir+json"
     },
-    "uri" : "http://localhost:8090/fhir/"
```

Comment:
No default but something is required.  Will be filled in by required assembly configuration property